modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/UserRow.es.js

```diff
@@ -36,6 +36,13 @@ export default withRouter(({creator = {}, statistics}) => (
 			</p>
 
 			<p className="c-mb-0 font-weight-bold text-dark">{creator.name}</p>
+
+			{companyMxName && (
+				<p>
+					{companyMxName[0]?.toUpperCase() +
+						companyMxName?.toLowerCase().substring(1).split('.', 1)}
+				</p>
```

Comment:
Let's simplify this;

```suggestion
				<p class="text-capitalize">
					{companyMxName}
				</p>
```

Reply:
It was necessary to keep ".split('.', 1)" because we need to remove from the point. for example just "Liferay" instead of "Liferay.com"